src/PluginForm/Bluesnap/PaymentMethodAddForm.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Drupal\commerce_bluesnap\PluginForm\Bluesnap;
+
+use Drupal\commerce\InlineFormManager;
+use Drupal\commerce_bluesnap\ApiService;
+use Drupal\commerce_payment\PluginForm\PaymentMethodAddForm as BasePaymentMethodAddForm;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * The BlueSnap PaymentMethodAddForm.
+ *
+ * @package Drupal\commerce_bluesnap\PluginForm\Bluesnap
+ */
+class PaymentMethodAddForm extends BasePaymentMethodAddForm {
+
+  /**
+   * The Bluesnap API helper service.
+   *
+   * @var \Drupal\commerce_bluesnap\ApiService
+   */
+  protected $apiService;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(
+    InlineFormManager $inline_form_manager,
```

Comment:
**CHANGE REQUEST**

So apparently you're building on top of Drupal Commerce `2.12+` as the inline form manager was recently introduced. This code will not work on sites using previous version. Update the module's `composer.json` accordingly here or in #3